server/sonar-server/src/test/java/org/sonar/server/issue/index/IssueIndexTest.java

```diff
@@ -621,6 +623,7 @@ public void facets_on_authors() {
     assertThat(result.getFacets().get("authors")).containsOnly(entry("steph", 1L), entry("simon", 2L));
   }
 
+  @Ignore
   @Test
   public void filter_by_created_after() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Fix or remove this skipped unit test [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1607)